### PR TITLE
set release for scratch build from build_id if release isn't set in D…

### PR DESCRIPTION
**Pull request by rcerven**
…ockerfile

Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: atomic_reactor/plugins/pre_bump_release.py [hunk 1]
```diff
@@ -137,6 +138,9 @@ def run(self):
 
         if self.append:
             next_release = self.get_next_release_append(component, version, release)
+        elif is_scratch_build():
```
Comment (MartinBasti) [inline atomic_reactor/plugins/pre_bump_release.py]: Shouldn't be scratch build first case to test to override other build options?

Comment (rcerven) [reply atomic_reactor/plugins/pre_bump_release.py]: that append will work even when "wrong" release is specified as it is just appending additional string to already existing release, so we can keep that functionality

Comment (MartinBasti) [reply atomic_reactor/plugins/pre_bump_release.py]: From my POW, scratch builds are isolated from any releases, so I assume that once --scratch is used, we shouldn't even touch koji for existing releases.

This makes results inconsistent.

File: atomic_reactor/plugins/pre_bump_release.py [hunk 2]
```diff
@@ -137,6 +138,9 @@ def run(self):
...
+            metadata = get_build_json().get("metadata", {})
+            next_release = metadata.get("name", "1")
```
Comment (MartinBasti) [inline atomic_reactor/plugins/pre_bump_release.py]: Haven't we talked about using a timestamp as default value?

Comment (rcerven) [reply atomic_reactor/plugins/pre_bump_release.py]: well we are using Openshift build name, if that isn't defined something is really off, so we just default to 1, that is really just corner case

**Comment by rcerven** [comment]
rn updated: 
- when release label isn't specified for scratch builds, openshift build id will be used as value

**Pull request merged**